webv2/profile/profile.go

```diff
@@ -201,6 +203,57 @@ func setConnectionProfile(isSource bool, sessionState session.SessionState, req
 
 }
 
+func CleanUpStreamingJobs(w http.ResponseWriter, r *http.Request) {
+	ctx := context.Background()
+	dsClient, err := datastream.NewClient(ctx)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("datastream client can not be created: %v", err), http.StatusBadRequest)
+	}
+	defer dsClient.Close()
+	sessionState := session.GetSessionState()
+	req := &datastreampb.DeleteStreamRequest{
+		Name: fmt.Sprintf("projects/%s/locations/%s/streams/%s", sessionState.GCPProjectID, sessionState.Region, sessionState.Conv.Audit.StreamingStats.DataStreamName),
+	}
+	op, err := dsClient.DeleteStream(ctx, req)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Error while deleting datastream job: %v", err), http.StatusBadRequest)
+		return
```

Comment:
If there is an error, we should not exit and continue trying to clean up other items. The final outcome should be a list/report of all the jobs cleaned up and those which errored out

Reply:
Can be taken up as a follow up

Reply:
Added item in the UI task list and will take it up later , as it is not a priority for the current release